google-cpi.html.md.erb

```diff
@@ -75,17 +78,25 @@ networks:
-* **on\_host\_maintenance** [String, optional]: [Instance behavior](https://cloud.google.com/compute/docs/instances/setting-instance-scheduling-options#onhostmaintenance) on infrastructure maintenance that may temporarily impact instance performance. Possible values: `MIGRATE` or `TERMINATE`. Default: `MIGRATE`.
-* **preemptible** [Boolean, optional]: If the VM should be [preemptible](https://cloud.google.com/preemptible-vms/). Default: `false`.
-* **service_scopes** [Array, optional]: [Authorization scope](https://cloud.google.com/docs/authentication#oauth_scopes) names for your default service account that determine the level of access your VM has to other Google services. Default: `[]` (no scope is assigned to the VM by default).
+| Option                  | Required | Type                                     | Example                                                                        | Description
+|:------------------------|:--------:|:-------------                            |:-------------                                                                  |:-----------
+| `machine_type`          | Y        | String                                   | `n1-standard-1`                                                                | The name of the [Google Compute Engine Machine Type](https://cloud.google.com/compute/docs/machine-types) the CPI will use when creating the instance (required if not using `cpu` and `ram`)
+| `cpu`                   | Y        | Integer                                  | `2`                                                                            | Number of vCPUs ([Google Compute Engine Custom Machine Types](https://cloud.google.com/custom-machine-types/)) the CPI will use when creating the instance (required if not using `machine_type`)
+| `ram`                   | Y        | Integer                                  | `2048`                                                                         | Amount of memory ([Google Compute Engine Custom Machine Types](https://cloud.google.com/custom-machine-types/)) the CPI will use when creating the instance (required if not using `machine_type`)
+| `zone`                  | N        | String                                   | `us-west1-a`                                                                   | The name of the [Google Compute Engine Zone](https://cloud.google.com/compute/docs/zones) where the instance must be created
+| `root_disk_size_gb`     | N        | Integer                                  | `10`                                                                           | The size (in Gb) of the instance root disk (default is `10Gb`)
```

Comment:
A default of `10Gb` would suggest to me that I need to include `Gb` suffix in the value, although I think it's supposed to be just the number.